src/intersection-observer-polyfill.js

```diff
@@ -429,6 +438,21 @@ export class IntersectionObserverPolyfill {
     return;
   }
 
+  /**
+   * Handle Mutation Observer Pass
+   * This performas the tick, and is wrapped in a paas
+   * To handle throttling of the observer
```

Comment:
Out of date.

Reply:
Is this still out of date? Since we agreed the pass is used in the comment below?